.circleci/config.yml

```diff
@@ -1375,3 +1389,50 @@ workflows:
       equal: [bump, << pipeline.parameters.action >>]
     jobs:
       - release-train
+
+  # to trigger tests manually, log into circleCI, select the project, a branch, and then click "Trigger Pipeline"
+  # and add a parameter called "run-manual-tests"
+  all-tests:
+    when:
+      and:
+        - not:
+            equal: [scheduled_pipeline, << pipeline.trigger_source >>]
+        - not: << pipeline.parameters.generate_snapshots >>
+        - not: << pipeline.parameters.generate_revenuecatui_snapshots >>
```

Comment:
I was wondering, do we need the above conditions? Or would the ones in the `or` be enough?

Reply:
good point, I didn't want to mess with existing conditions but this would never happen if it's a separate workflow